data/git-datasets.md

```diff
@@ -106,15 +95,13 @@ While you're logged in to the web interface:
 
 ## Listing the available datasets
```

Comment:
Right now new users aren't auto-added to the datasets org and need to be added to this org by an admin. This should maybe be mentioned somewhere in case this doesn't happen during onboarding, so they know who to request access from.

Reply:
Thanks for the suggestion! I added a note (and a helpful link for the admins who have to do it).